src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,69 @@
+
+package duke;
+
+import duke.tasks.Deadline;
+import duke.tasks.Events;
+import duke.tasks.Task;
+import duke.tasks.TaskList;
+import duke.tasks.Todo;
+import java.util.Scanner;
+
+public class Ui {
+    private Scanner sc = new Scanner(System.in);
+
+    public void hello() {
+        System.out.println("__________________________________________________________");
```

Comment:
Since the horizontal line is used frequently in our code, perpaps it will be better to declare it as a constant?